src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command.py

```diff
@@ -133,7 +133,7 @@ def _cmd_step(self, py_db, cmd_id, seq, text):
 
     def _cmd_set_next(self, py_db, cmd_id, seq, text):
         thread_id, line, func_name = text.split('\t', 2)
-        return self.api.request_set_next(py_db, thread_id, cmd_id, line, func_name)
+        return self.api.request_set_next(py_db, 0, thread_id, cmd_id, line, func_name)
```

Comment:
The `seq` is passed as a parameter in this function, so, you could pass the proper `seq` in this case instead of `0`.